package.json

```diff
@@ -61,6 +61,7 @@
         "knockout-sortable": "1.2.2",
         "knockout-validation": "Knockout-Contrib/Knockout-Validation#2.0.4",
         "leaflet": "0.7.7",
+        "leaflet-fullscreen": "^1.0.2",
```

Comment:
Because we use `mapbox.js` for the map in web apps, not the version of leaflet here in `package.json`, it could also make sense to use their CDN instead of adding a dependency: https://docs.mapbox.com/mapbox.js/plugins/#leaflet-fullscreen

Reply:
Yes, both of those maps in web apps use the mapbox.js implementation of leaflet. It looks like we use this dependency leaflet in reports.

Reply:
Adding the dependency here is good. Our prod files are bundled and served from our own CDN, so using mapbox's CDN doesn't gain us anything.